gallery_app/lib/widgets/row_column_expanded_widgets/row_column_menu.dart

```diff
@@ -0,0 +1,183 @@
+
+class RowColumnMenu extends StatelessWidget {
+  const RowColumnMenu({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final RowColumnCubit rowColumnCubit =
+        BlocProvider.of<RowColumnCubit>(context);
+    return BlocBuilder<RowColumnCubit, RowColumnState>(
+      builder: (context, state) {
+        return Container(
+          padding: const EdgeInsets.all(10),
+          color: Colors.white,
+          child: Column(
+            children: [
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  Row(
+                    children: [
+                      Radio(
+                        activeColor: Colors.blue,
+                        value: "Row",
+                        groupValue: state.type,
+                        onChanged: (value) {
+                          rowColumnCubit.setType(value.toString());
+                        },
+                      ),
+                      const Text("Row")
+                    ],
+                  ),
+                  Row(
+                    children: [
+                      Radio(
+                        activeColor: Colors.blue,
+                        value: "Column",
+                        groupValue: state.type,
+                        onChanged: (value) {
+                          rowColumnCubit.setType(value.toString());
+                        },
+                      ),
+                      const Text("Column")
+                    ],
+                  ),
+                ],
+              ),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  const Text(
+                    "mainAxisSize",
+                    style: TextStyle(fontSize: 18),
+                  ),
+                  DropdownButton(
+                    value: state.mainAxisSize,
+                    items: MainAxisSize.values.map((value) {
+                      return DropdownMenuItem(
+                        value: value,
+                        child: Text(value.toString().split('.')[1]),
+                      );
+                    }).toList(),
+                    onChanged: (value) {
+                      rowColumnCubit.setMainAxisSize(value);
+                    },
+                  )
+                ],
+              ),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  const Text(
+                    "mainAxisAlignment",
+                    style: TextStyle(fontSize: 18),
+                  ),
+                  DropdownButton(
+                    value: state.mainAxisAlignment,
+                    items: MainAxisAlignment.values.map((value) {
+                      return DropdownMenuItem(
+                        value: value,
+                        child: Text(value.toString().split('.')[1]),
```

Comment:
Please take note, enum variables have a default getter to return returns its name, for example:
> MainAxisAlignment.spaceBetween.name => "spaceBetween"